.github/workflows/pypi.yml

```diff
@@ -26,6 +26,8 @@ jobs:
         fail_ci_if_error: true
         files: ./coverage.xml
         flags: unittests
+        project: off
+        patch: off
```

Comment:
Can you give more desc for this changes, so that I can easily understand what you have done ?  and I notice that the latest version of `coverage` is V3, can you upgrade it ? If so, it will be more great for this contribution.

Reply:
I notice that if coverage decrease, `x` will be show in commit. like this: https://github.com/wechaty/python-wechaty/commit/694057719a52da7a5e07d50d1c5935f3411bd241  So, turn off is better.
![163317103-b3c3510d-8f5a-4ab3-bf27-a8da93a99e81](https://user-images.githubusercontent.com/4636271/178097827-7ad7cb88-d2f2-4b6d-9285-b083b9ea7622.png)
However, I will not change it this patch.